src/ica.jl

```diff
@@ -88,6 +88,7 @@ function fastica!(W::DenseMatrix{Float64},      # initialized component matrix,
     end
 
     # main loop
+    chg = 0.0
```

Comment:
Unused variable?

Reply:
Will change it to `NaN` but it makes chg available outside scope of the loop & also (when `NaN` covers the case of zero iterations.

Reply:
Got it, thanks!